tasks/bin/cross-ruby.rake

```diff
@@ -130,6 +130,9 @@ end
 # make
 file "#{build_dir}/ruby.exe" => ["#{build_dir}/Makefile"] do |t|
   chdir File.dirname(t.prerequisites.first) do
+    # From https://github.com/grpc/grpc/issues/12161. Old rubies have race conditions around verconf.h.
```

Comment:
Which versions are "Old rubies"?

Reply:
I looked into it as I didn't know the details. It turned out the "old rubies" here are 2.3.x or before. Added a comment about it.

Reply:
2.3.0 is quite old to me, but protocolbuffers/protobuf still seems to provide builds for them https://github.com/protocolbuffers/protobuf/blob/v3.14.0/ruby/Rakefile#L84

Reply:
OK.
We don't support EOE-ed Rubies.

Reply:
I see. Thanks anyway for reviewing this!